src/stackwalk.c

```diff
@@ -1160,6 +1161,17 @@ JL_DLLEXPORT void jl_print_task_backtraces(void) JL_NOTSAFEPOINT
     }
     jl_safe_printf("==== Done\n");
 }
+// Print backtraces for all live tasks, for all threads.
+// WARNING: this is dangerous and can crash if used outside of gdb, if
+// all of Julia's threads are not stopped!
+JL_DLLEXPORT void jl_print_task_backtraces(void) JL_NOTSAFEPOINT
+{
+    _jl_print_task_backtraces(0);
+}
+JL_DLLEXPORT void jl_print_task_backtraces_skip_done(void) JL_NOTSAFEPOINT
+{
+    _jl_print_task_backtraces(1);
+}
```

Comment:
One question I have on this PR is:
- Since there hasn't been a version of Julia released yet with `jl_print_task_backtraces(void)`, I guess it's not too late to make backwards-incompatible changes.
   Should we just change the API to be `jl_print_task_backtraces(bool)` instead of adding another function? (of course actually `int`, not `bool`, because `C`.)

Reply:
Okay actually, if we want to do the other change suggested in https://github.com/JuliaLang/julia/issues/47932 (allowing it to optionally write to a file), i don't want to have the explosion of functions from those 2 options. So maybe these should just be parameters, and we should just change the API. 👍

Reply:
If only C allowed for optional params though :'(

Reply:
I agree that adding a parameter and changing the API is the simplest path.

Reply:
Prefer a change to the API as well

Reply:
K thx, done